Jint.Tests.Test262/Test262Test.cs

```diff
@@ -290,6 +291,48 @@ public static IEnumerable<object[]> SourceFiles(string pathPrefix, bool skipped)
                     reason = "Unicode support and its special cases need more work";
                 }
 
+                if (name.StartsWith("language/statements/class/subclass/builtin-objects/Promise"))
+                {
+                    skip = true;
+                    reason = "Promise not implemented";
```

Comment:
WAT?

Reply:
Just a small PR and you can enable this... Just need to have the constructor logic use correct methods, like I mentioned 😉